tests/data/tagged_codelist/foo_attr_list.yaml

```diff
@@ -0,0 +1,16 @@
+- Emissions|Species:
+    # the "Sector" guards against feature regression (conflict with "Sector" tag)
+    definition: Emissions of Species
+    unit: tSpecies/yr
+
```

Comment:
This feature regression is already tested in `foo.yaml` so suggest to remove here for simplicity.
```suggestion
```